ddtrace/tracer/span_test.go

```diff
@@ -235,25 +236,57 @@ func TestSpanString(t *testing.T) {
 }
 
 func TestSpanSetMetric(t *testing.T) {
-	assert := assert.New(t)
-	tracer := newTracer(withTransport(newDefaultTransport()))
-	span := tracer.newRootSpan("pylons.request", "pylons", "/")
-
-	// check the map is properly initialized
-	span.SetTag("bytes", 1024.42)
-	assert.Equal(3, len(span.Metrics))
-	assert.Equal(1024.42, span.Metrics["bytes"])
-	_, ok := span.Metrics[keySamplingPriority]
-	assert.True(ok)
-	_, ok = span.Metrics[keySamplingPriorityRate]
-	assert.True(ok)
-
-	// operating on a finished span is a no-op
-	span.Finish()
-	span.SetTag("finished.test", 1337)
-	assert.Equal(3, len(span.Metrics))
-	_, ok = span.Metrics["finished.test"]
-	assert.False(ok)
+	const intUpperLimit = int64(1) << 53
+	const intLowerLimit = -intUpperLimit
```

Comment:
These are repeated too much, we should maybe declare them globally?
```go
const (
    // maxIntInFloat64 specifies the maximum integer that can be fitted inside a float64 without loss of precision.
    maxIntInFloat64 = int64(1) << 53
    // ...
    minIntInFloat64 = -maxIntInFloat64
)
```
Or something like that....

Reply:
👍 Ok. We don't want to use them in both code *and* tests though. I'll keep one global set for tests.

Reply:
Why not? We _do_ want to use them in both code and tests. We don't want code to diverge from tests and end up having tests pass for broken code. They should be defined only in one place. And I'd prefer it if we didn't call them `intUpperLimit` because it's not the `int` upper limit, it's just the maximum `int` that fits in a `float64`.

Reply:
Let's ignore this comment. @labbati made me aware of my mistake in reasoning about this in https://github.com/DataDog/dd-trace-go/pull/542#discussion_r355903413. I think these constants can go back inside the test since we don't need them globally anymore if they are used only in one test, right?